components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,86 @@
+@import "@not-govuk/sass-base";
+@import "govuk-frontend/govuk/helpers/_colour.scss";
+
+.hods-timeline {
+  overflow: hidden;
+  position: relative;
+
+  p {
+    text-decoration: inherit;
+    font-size: inherit;
+  }
+
+  &__item {
+    @include govuk-responsive-padding(2, "bottom");
+    @include govuk-responsive-padding(5, "left");
+    /* padding size 6 looks better with smaller viewports. Can media queries
+    be used here? */
+    position: relative;
+
+    &:hover {
+      background-color: #E3E3E3;
+    }
+
+    &:before {
+      background-color: #1d70b8;
+      content: "";
+      @include govuk-responsive-padding(1, "top"); /* size of blue lines */
+      @include govuk-responsive-padding(3, "right");
+      left: 0;
+      position: absolute;
+      @include govuk-responsive-margin(1, "top");
+
+      z-index: 1;
+    }
+  }
+
+  &:before {
+    background-color: #1d70b8;
+    content: "";
+    height: 100%;
+    left: 0;
+    position: absolute;
+    @include govuk-responsive-margin(1, "top");
+    /* this is more restricive than using px. The ideal margin is around 1.5
+    using this govuk element, but that doesnt exist. I can only use either 1 or
+    2.  1 looks better when the viewport is narrow, and 2 looks better when the
+    viewport is wide. Can @media queries be used here?
+    mobile:  320px,
+    tablet:  641px,
+    desktop: 769px
+    */
```

Comment:
I don't think this comment should be committed. I don't think it adds much value to future programmers.

The purpose of the restrictiveness is to aid consistency in the design. We can break out of that if we really need to but it makes things more complicated.

The mixin will use media queries under the hood. I think GDS also provide some helpers for taking greater control. You could look at the implementation of the `govuk-responsive-padding` mixin to see how if you thought that was required.

It might be good to see some screenshots.